resources/sass/global/_variables.scss

```diff
@@ -24,7 +22,40 @@ $orange: #f6993f;
 $yellow: #ffed4a;
 $green: #38c172;
 $teal: #4dc0b5;
-$cyan: #6cb2eb;
-$i-white: #FFFFFF;
-$i-blue: #2699FB;
-$primary: $i-blue;
+
+$primaryColor: rgb(38,153,251);
+$primaryColor--light: lighten($primaryColor, 0.6);
+$primaryColor--med: rgb(16, 109, 188);
```

Comment:
I think --med should be expanded to medium or use another name for it.

Reply:
coo